autoload/AnsiEsc.vim

```diff
@@ -2212,16 +2206,10 @@ fun! s:MultiElementHandler()
 
     " build highlighting rule
     let hirule= "hi ansiMEH".mehcnt
-    if has("gui") && has("gui_running")
-     let hirule=hirule." gui=".mod
-     if fg != ""| let hirule=hirule." guifg=".fg| endif
-     if bg != ""| let hirule=hirule." guibg=".bg| endif
-    else
-     let hirule=hirule." cterm=".mod
-     if fg != ""| let hirule=hirule." ctermfg=".fg| endif
-     if bg != ""| let hirule=hirule." ctermbg=".bg| endif
-    endif
-"    call Decho(" exe hirule: ".hirule)
+    let hirule=hirule." cterm=".mod." gui=".mod
+    if fg != ""| let hirule=hirule." ctermfg=".fg." guifg=".guifg| endif
```

Comment:
Probably it makes sense to check both (`fg` OR `guifg` is set), and same on next line.